tests/fixtures/fixture_users.py

```diff
@@ -28,6 +28,7 @@ def build_unique_user(group: str, api_client: requests) -> utils.TestUser:
         _group_id=user_data.get("groupId"),
         user_id=user_data.get("id"),
         email=user_data.get("email"),
+        username=user_data.get("username"),
         token=token,
     )
 
```

Comment:
The TestUser is created in multiple places you'll need to add this in a few more places.

Reply:
copy that.